server/controllers/payment.js

```diff
@@ -82,14 +82,12 @@ exports.payPetSitter = asyncHandler(async (req, res) => {
 
 });
 
-exports.getStripePk = asyncHandler(async (req, res) => {
-  const user = await Profile.findById(req.params.id);
+exports.getStripePK = (req, res) => {
```

Comment:
Not entirely sure what this endpoint is for 🤔 Also, I think it would be a better idea to name this more explicitly just so we don't create a confusion - both `Public/Publishable` and `Private` starts with a `P` 🙃

Reply:
Hi Ugur. When I started this issue I was trying to retrieve the publishable key from the BE. But I'm not using it, I'll remove it. Thanks.